core/flyout.js

```diff
@@ -730,6 +810,11 @@ Blockly.Flyout.prototype.addBlockListeners_ = function(root, block, rect) {
       block.addSelect));
   this.listeners_.push(Blockly.bindEvent_(rect, 'mouseout', block,
       block.removeSelect));
+
+  if (block.flyoutCheckbox) {
+    this.listeners_.push(Blockly.bindEvent_(block.flyoutCheckbox, 'mousedown', null,
```

Comment:
What about them?  This is the same way that we add listeners on the blocks in the flyout.